spec/validators/fee/base_fee_validator_spec.rb

```diff
@@ -262,7 +271,7 @@
         [ppe_fee, npw_fee].each do |f|
           f.rate = 25
           expect(f).to_not be_valid
-          expect(f.errors[:rate]).to include("#{f.fee_type.code.downcase}_must_be_blank")
+          expect(f.errors[:rate]).to include(match(/fees must not have a rate/))
```

Comment:
_Suggestion_: Matcher improvements

`.* fees must not have a rate`
OR
`(.*?)(Pages of prosecution evidence|Number of prosecution witnesses) fees must not have a rate`